provisioning/service/samples/createCertificate.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+var pem = require('pem');
+var fs = require('fs');
+var certPath = __dirname + "\\cert\\";
```

Comment:
>__dirname + "\\cert\\"; [](start = 15, length = 23)

I would encourage using path.join or path.resolve instead to automatically deal with cross-platform issues.